Formula/libtiff.rb

```diff
@@ -15,8 +15,10 @@ class Libtiff < Formula
 
   option :universal
   option :cxx11
+  option "with-lzma", "Include support for LZMA compression"
```

Comment:
I considered that, but the libtiff option is actually `lzma`, which is what I'd be looking for if I were installing lzma. The fact this is packaged as `xv` in Homebrew seems arbitrary. Are you sure?